### PR TITLE
adding separators to README file

### PR DESCRIPTION
**Pull request by ghost**
**in this PR i did basically:**

- *add separators to README file;*

---

**I did this because:**

- *i feel it improves the overall UI/UX of the file;*
- *it makes the file easier to be read;*
- *it makes the file more pleasent to be read;*
- *it makes the file more easier to be understood;*

### CODE REVIEW

**Review by MichaelKohler — CHANGES_REQUESTED** [review]
Thanks! Unfortunately we have a conflict here as well, as Git can't merge this one automatically after merging the other Pull Request.

**Comment by ghost** [comment]
DONE!!! I have already learned something about git with you today @MichaelKohler thank you so much for helping me to improve my coding skills. cheers 👍

**Comment by MichaelKohler** [comment]
Looks like something has gone wrong, I can't find the change on the separator anymore for the README. Additionally, you also now have the changes (meant for your PR #20) in this one as well. No worries though, I've applied those changes:

https://github.com/mozilla/fx-webcompat-campaign/commit/b1fc3ad16e5c12809845132ce3b1a687ee56753f
https://github.com/mozilla/fx-webcompat-campaign/commit/d8042fe09187c6e71f0902147a6c2d07433fe371

Everything with the right attribution :)

Unfortunately I can't quite say what went wrong on the rebase. It might be worth looking into your bash history to figure out where it went wrong. If you want to, you can send me the output of "history" and I'll have a look to better tell you :)

Thanks for your contributions, as those two things are merged now manually, I'll close this PR.

**Pull request closed without merging**
